tests/local_onionshare_settings_dialog_legacy_tor_test.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+import json
+import unittest
+import time
+from PyQt5 import QtCore, QtTest
```

Comment:
Minor nitpick: the import of `json`, `time`, and the `Qt*` modules are not needed in these new tests - only `unittest`

Reply:
Good catch, I removed the unneeded imports